api/tests/unit/organisations/test_unit_organisations_models.py

```diff
@@ -29,72 +27,63 @@
 )
 from organisations.subscriptions.metadata import BaseSubscriptionMetadata
 from organisations.subscriptions.xero.metadata import XeroSubscriptionMetadata
-from users.models import FFAdminUser
-
-
-@pytest.mark.django_db
-class OrganisationTestCase(TestCase):
-    def test_can_create_organisation_with_and_without_webhook_notification_email(self):
-        organisation_1 = Organisation.objects.create(name="Test org")
-        organisation_2 = Organisation.objects.create(
-            name="Test org with webhook email",
-            webhook_notification_email="test@org.com",
-        )
-
-        self.assertTrue(organisation_1.name)
-        self.assertTrue(organisation_2.name)
-
-    def test_has_paid_subscription_true(self):
-        # Given
-        organisation = Organisation.objects.create(name="Test org")
-        Subscription.objects.filter(organisation=organisation).update(
-            subscription_id="subscription_id"
-        )
-
-        # refresh organisation to load subscription
-        organisation.refresh_from_db()
-
-        # Then
-        assert organisation.has_paid_subscription()
-
-    def test_has_paid_subscription_missing_subscription_id(self):
-        # Given
-        organisation = Organisation.objects.create(name="Test org")
-
-        # Then
-        assert not organisation.has_paid_subscription()
-
-    @mock.patch("organisations.models.cancel_chargebee_subscription")
-    def test_cancel_subscription_cancels_chargebee_subscription(
-        self, mocked_cancel_chargebee_subscription
-    ):
-        # Given
-        organisation = Organisation.objects.create(name="Test org")
-        user = FFAdminUser.objects.create(email="test@example.com")
-        user.add_organisation(organisation, role=OrganisationRole.ADMIN)
-        Subscription.objects.filter(organisation=organisation).update(
-            subscription_id="subscription_id", payment_method=CHARGEBEE
-        )
-        subscription = Subscription.objects.get(organisation=organisation)
-
-        # refresh organisation to load subscription
-        organisation.refresh_from_db()
-
-        # When
-        organisation.cancel_subscription()
-
-        # Then
-        mocked_cancel_chargebee_subscription.assert_called_once_with(
-            subscription.subscription_id
-        )
-        # refresh subscription object
-        subscription.refresh_from_db()
-        # Subscription has been immediately transformed to free.
-        assert subscription.cancellation_date is None
-        assert subscription.subscription_id is None
-        assert subscription.billing_status is None
-        assert subscription.payment_method is None
-        assert subscription.plan == FREE_PLAN_ID
+
+
+def test_has_paid_subscription_true(db: None) -> None:
```

Comment:
Can we update the test name to add the name of the class as well... something like `test_organisation...` it looks a bit confusing otherwise

Reply:
Sure I updated that and other tests in the module.